src/settings/hotkeysSettings.js

```diff
@@ -0,0 +1,85 @@
+import store from '@/store/index';
+import { shuffleArray } from '@/utils/utils';
+
+export const initHotkeys = function initHotkeys() {
+  document.addEventListener('keyup', event => {
+    const key = event.code;
+
+    if (!event.target.closest('input[type=text]') && !event.target.closest('textarea')) {
+      createNewItem(key);
+      switchEdittingItemBack(key);
+      createNewList(key);
+      updateCurrentList(key);
+      randomizeList(key);
+      switchFocusMode(key);
+      exitFocusMode(key);
+      switchSidebarMode(key);
+    }
```

Comment:
hmmm. Here we go inside every function every time. I think it's better to check here in switch case loop the key and call exactly function just when the condition is working.